packages/lesswrong/server/debouncer.ts

```diff
@@ -277,10 +278,14 @@ export const forcePendingEvents = async (upToDate=null) => {
     
     if (eventToHandle) {
       await dispatchEvent(eventToHandle);
+      countHandled++;
     }
     
     // Keep checking for more events to handle so long as one was handled.
   } while (eventToHandle);
+
+  // eslint-disable-next-line no-console
+  console.log(`Forced ${countHandled} pending event${countHandled === 1 ? "" : "s"}`);
```

Comment:
Again, after discussion with JP, we agreed this utility should have some kind of console feedback.